db/seeds/schemas/clean_lgfs_fee_scheme_10.rb

```diff
@@ -0,0 +1,163 @@
+          begin
+            scheme_nine_offence_for(offence)
+            puts '    Scheme 9 offence exists:      Yes'
+          rescue MissingSchemeNineOffence
+            puts '    Scheme 9 offence exists:      No'
+          end
+          puts "    Claim ids:"
+          offence.claims.each do |claim|
+            puts "      #{claim.id}"
+          end
+        end
+      end
+
+      def up
+        lgfs_scheme_ten_only.each do |offence|
+          puts "Offence: #{offence.unique_code}"
+          puts "  #{offence.description[0, 60]}"
+          puts "  #{offence.offence_class.description[0, 60]}"
+          Offence.transaction do
+            new_offence = scheme_nine_offence_for(offence)
+            update_claims(offence.claims, new_offence)
+            add_scheme_ten_to(new_offence)
+            remove_redundant(offence)
+          end
+          puts "-----"
+        end
+      end
+
+      def down
+        all_lgfs_offences.each do |offence|
+          puts "Offence: #{offence.unique_code}"
+          puts "  #{offence.description[0, 60]}"
+          puts "  #{offence.offence_class.description[0, 60]}"
+          Offence.transaction do
+            create_scheme_ten_offence_for(offence)
+          end
+          puts "-----"
+        end
+      end
+
+      private
+
+      def fee_scheme_nine = @fee_scheme_nine ||= FeeScheme.lgfs.nine.first
+      def fee_scheme_ten = @fee_scheme_ten ||= FeeScheme.lgfs.ten.first
+
+      def all_lgfs_offences = @all_lgfs_offences ||= Offence.joins(:fee_schemes).merge(FeeScheme.lgfs).distinct
+      def lgfs_scheme_nine_only = all_lgfs_offences.select { |o| o.fee_schemes & [fee_scheme_nine, fee_scheme_ten] == [fee_scheme_nine] }
+      def lgfs_scheme_ten_only = all_lgfs_offences.select { |o| o.fee_schemes & [fee_scheme_nine, fee_scheme_ten] == [fee_scheme_ten] }
+      def lgfs_scheme_nine_or_ten = all_lgfs_offences.select { |o| o.fee_schemes & [fee_scheme_nine, fee_scheme_ten] == [fee_scheme_nine, fee_scheme_ten] }
+
+      def lgfs_scheme_ten_only_with_claims = lgfs_scheme_ten_only.select { |o| o.claims.count.positive? }
+
+      def scheme_nine_offence_for(offence)
+        code = offence.unique_code.gsub(/~.*$/, '')
+        Offence.find_by(unique_code: code).tap do |new_offence|
+          raise MissingSchemeNineOffence unless offences_match?(offence, new_offence)
+        end
+      end
+
+      def create_scheme_ten_offence_for(offence)
+        if offence.unique_code.match(/~10/)
+          puts "    [NOT-DUPLICATING] Offence #{offence.unique_code}".yellow
+          return
+        end
+        begin
+          new_offence = offence.dup
+          new_offence.id = 7999 + offence.id # lib/tasks/lgfs_scheme_ten.rake starts adding ids at 8000
+          new_offence.unique_code = new_offence.unique_code + '~10'
+          new_offence.fee_schemes = [fee_scheme_ten]
+          # claims = offence.claims.select { |claim| claim.fee_scheme == fee_scheme_ten }
```

Comment:
Can this and the other comments in this file be removed?

Reply:
I was in two minds about this. The commented sections are what are required to set the offences of fee scheme 10 claims but it is very slow if there are lots of claims.